src/main/java/gov/cabinetofice/gapuserservice/service/user/OneLoginUserService.java

```diff
@@ -166,4 +176,18 @@ public void invalidateUserJwt(final Cookie customJWTCookie, final HttpServletRes
         thirdPartyAuthToken.setMaxAge(0);
         response.addCookie(thirdPartyAuthToken);
     }
+
+    public void validateRoles(List<Role> dbRoles, String roles) {
```

Comment:
Same here for the naming of dbRoles and roles. 

It should be clearer that one is the user associated roles stored in the db and the other is roles associated with the payload